src/references/index.js

```diff
@@ -72,6 +73,16 @@ export const tokenOverrides = mapKeys(tokenOverridesData, (_, address) =>
   toLower(address)
 );
 
+const loweredUniswapPairs = mapKeys(uniswapPairsData, (value, key) =>
```

Comment:
This file "uniswap-pairs.json" as it is now is going to be obsolete and replaced by our rainbow tokens list once they are integrated (also the temporary hardcoded token list exists in references/uniswap/index.js as `CURATED_UNISWAP_TOKENS`)

Reply:
I'm going to approve and we can merge for now, and fix this up later once rainbow token list is integrated fully